pkg/pool-stable/contracts/StablePool.sol

```diff
@@ -726,8 +726,7 @@ contract StablePool is BaseGeneralPool, BaseMinimalSwapInfoPool, StableMath, IRa
     }
 
     function _setAmplificationData(uint256 value) private {
-        _setAmplificationData(value, value, block.timestamp, block.timestamp);
-
+        _storeAmplificationData(value, value, block.timestamp, block.timestamp);
         emit AmpUpdateStopped(value);
```

Comment:
Not a big deal, but with this change we are emitting only `AmpUpdateStopped` when setting amp data and no start event.

Reply:
Yes, I think that is perfectly fine. This was noted in the audit report.